include/TMS/dev/TMP117.hpp

```diff
@@ -36,7 +40,7 @@ class TMP117 {
     /**device ID*/
     uint8_t deviceID;
     /**i2c declaration*/
```

Comment:
Add a comment explaining why this had to be a pointer instead of a reference. I don't remember what the exact error was, but you can change this in your code to get the error to fire again.